### PR TITLE
PCHR-2035: Variables used for select dropdown

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
Variables are used in shoreditch(https://github.com/compucorp/org.civicrm.shoreditch/pull/19) for `height` and `border-color`, so this repository need to be updated as it imports partials from Shoreditch.

## Technical Details
`$input-border` variable set to `$crm-color-neutral` as it is the default border color is this theme.
Also `$crm-color-neutral` is replaced by `$input-border` wherever it was used for border.

## Comment
Can only be merged once, https://github.com/compucorp/org.civicrm.shoreditch/pull/19 is merged and 1.7 is synced.

### CODE REVIEW

File: civihr_default_theme/scss/base/_variables.scss [hunk 1]
```diff
@@ -163,3 +163,8 @@ $crm-scrollbar-thumb-color: $crm-gray-light-blue !default;
 $crm-customselect-height:        33 !default;
 $crm-customselect-caret-width:   32 !default;
 $crm-customselect-padding-right: $crm-customselect-caret-width + 12 !default;
+
+//== Forms
+//
+//##
+$input-border:              $crm-color-neutral !default;
```
Comment (AkA84) [inline civihr_default_theme/scss/base/_variables.scss:170]: is `$input-border` a default bootstrap SASS variable? If yes, what impact does this have on the SSP theme as a whole?

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/base/_variables.scss (hunk 1 above)
Comment (deb1990) [reply civihr_default_theme/scss/base/_variables.scss:170]: Yes it is bootstrap, other components were using `$crm-color-neutral` for the border, so replaced it with `$input-border`. Added in PR document too.

Comment (AkA84) [reply civihr_default_theme/scss/base/_variables.scss:170]: so you defined the value of a default bootstrap variable, shouldn't have an effect on the entire theme? (i have to assume that `$input-border`, being a default variable, is used across several SCSS partials)

Comment (deb1990) [reply civihr_default_theme/scss/base/_variables.scss:170]: I have checked that `$input-border`is not used anywhere else other than the `form-control`, also `$crm-color-neutral` was used previously for defining the border which was wrong. So using `input-border` makes sense as it internally is same as `$crm-color-neutral`.

**Review by AkA84 — APPROVED** [review]

**Pull request merged**